src/optics/lib/Convex/Lenses.hs

```diff
@@ -0,0 +1,552 @@
+{-# LANGUAGE DataKinds           #-}
+{-# LANGUAGE GADTs               #-}
+{-# LANGUAGE LambdaCase          #-}
+{-# LANGUAGE NamedFieldPuns      #-}
+{-# LANGUAGE RankNTypes          #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeApplications    #-}
+{-| Lenses for @cardano-api@ types
+-}
+module Convex.CardanoApi.Lenses(
```

Comment:
Shouldn't this be `Convex.Lenses`?

Reply:
Oh right! I'll use that module to re-export all the optics, which name do you prefer:
- `Convex.Lenses`
- `Convex.Optics`

?

Reply:
Optics (it's the more general one)

Reply:
I've removed this file but GH isn't realizing it